src/index.ts

```diff
@@ -8,14 +8,15 @@ import rateLimit from './helpers/rateLimit';
 import log from './helpers/log';
 import initMetrics from './helpers/metrics';
 import { checkKeycard } from './helpers/keycard';
-import './helpers/moderation';
+import refreshModerationData from './helpers/moderation';
 import './helpers/strategies';
 
 const app = express();
 const PORT = process.env.PORT || 3000;
 
 initLogger(app);
 initMetrics(app);
+refreshModerationData();
```

Comment:
Ideally we should avoid other changes on PRs with different context, 😄 when i look at PR it contains 
![image](https://github.com/snapshot-labs/snapshot-hub/assets/15967809/55ba20d9-ff44-4220-9c1f-4ee507211c49)

It scares me, so i add it to my todo and get delayed 🙈

Reply:
PR is less scary now, majority of changes are tests and config files